src/main/java/com/solvdLaba/airport/Passenger.java

```diff
@@ -10,32 +11,30 @@ public class Passenger implements Security{
     private boolean isTSAChecked;
     private Boarding boardingPass;
 
-    public Luggage getLuggage() {
-        return luggage;
-    }
-
-    public void setName(String name) {
+    public Passenger(String name, int age){
         this.name = name;
-    }
-
-    public void setAge(int age) throws InvalidAgeException {
         this.age = age;
-        if (age < 0){
-            throw new InvalidAgeException("Age cannot be less than zero", new IllegalArgumentException() {
-            });
-        }
-
+        MyLogger.info("Passenger getting created:  " + name);
     }
-
-    public void setLuggage(Luggage luggage) {
-        this.luggage = luggage;
+    public boolean isTSAChecked() {
+        return isTSAChecked;
     }
 
 
-    public Passenger(String name, int age){
-        this.name = name;
-        this.age = age;
-      //  MyLogger.info("Passenger getting created:  " + name);
+    protected boolean hasValidBoardingPass(Passenger p1) throws InvalidBoardingPassException {
```

Comment:
So lets summarize:
if boarding pass is valid: true
if boarding pass is invalid: Excepetion

Imaging an airport staff checks boarding pass of a customer, and in case boarding pass is invalid -> application crashes or sends an error. Is it the behaviour that we expect?

Reply:
After your comment I found this method doesn't have the logic. That's why I deleted it.